geoportal/c2cgeoportal_geoportal/scaffolds/create/qgisserver/geomapfish.yaml_tmpl

```diff
@@ -3,11 +3,18 @@ vars:
   schema: '{PGSCHEMA}'
   schema_static: '{PGSCHEMA_STATIC}'
   sqlalchemy.url: postgresql://{PGUSER}:{PGPASSWORD}@{PGHOST}:{PGPORT}/{PGDATABASE}
```

Comment:
It seems `sqlalchemy.url` is not used anymore

Reply:
Probably yes but I like to keep it...